docs/section10/10-14.md

```diff
@@ -0,0 +1,189 @@
+                - Set heating capacity equal to the rated heating capacity: `total_heat_capacity_b = heating_system_b.rated_total_heat_capacity`
+            - Else if, check that design heating capacity does not equal null:`Elif heating_system_b.design_total_heat_capacity != Null:`  
+                - Set heating capacity equal to the design heating capacity: `total_heat_capacity_b = heating_system_b.design_total_heat_capacity`
+            - Else, set the was_heat_capacity_not_defined boolean variable equal to TRUE: `Else: was_heat_capacity_not_defined = TRUE`
+            - Check if capacity is <225,000 Btuh (may need an error handler in case capacity is not defined) or if was_heat_capacity_not_defined boolean variable equals TRUE (below checks if the modeled value is equal to the most stringent size category which is <225,000 for when capacity is not defined) or [if the aggregation factor was defined (i.e., does not equal Null) and the resultant capacity is less than 225000]: `if total_heat_capacity_b < 225000 OR was_heat_capacity_not_defined boolean variable == TRUE OR (zone_list_b[0].aggregation_factor != Null AND total_heating_capacity_b/zone_list_b[0].aggregation_factor < 225000):`  
+                - Loop through each of the efficiency_metric_types associated with the heating system until AFUE is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                    - Check if the efficiency metric equals ANNUAL_FUEL_UTILIZATION_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "ANNUAL_FUEL_UTILIZATION_EFFICIENCY":` 
+                        - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`
+                        - Set the efficiency found boolean variable to TRUE: `was_efficiency_found = TRUE`
+                    - Else, do nothing and move on to the next metric: `Else:`   
+                - Check if was_efficiency_found equals TRUE: `if was_efficiency_found == TRUE:`  
+                    - Set expected_baseline_eff_b equal to 78% AFUE: `expected_baseline_eff_b = 0.78`
+                    - Check if the modeled efficiency meets the expected baseline efficiency by using the RCT compare_standard_val function, Permit_More_Stringent_Value is defined globally and is TRUE OR FALSE: `efficiency_matches_acceptable_value = compare_standard_val(Permit_More_Stringent_Value, expected_baseline_eff_b, modeled_efficiency_b, "greater than")`  func shall return True or False, "greater than" means that if the modeled_efficiency_b is greater than or equal to the expected_baseline_eff_b it would be accepted if Permit_More_Stringent_Value equals TRUE                  
+                - Reset was_efficiency_found equal to FALSE: `was_efficiency_found = FALSE`
```

Comment:
@mkarpman It can be one or the other but if both are included they should both be correct based on this logic (I think).

Reply:
From call if both are defined and one is correct then accept

Reply:
Implemented the following: . For furnaces <225000 Btuh where the baseline efficiency can be modeled as 78% AFUE or 80% Et, if both are defined under HeatingSystem since efficiency can be defined in multiple units, we pass it if the project meets at least one of the 2 values in App G.